server/src/main/java/io/spine/system/server/SystemSettings.java

```diff
@@ -190,12 +207,12 @@ public boolean equals(Object o) {
         var settings = (SystemSettings) o;
         return commandLog == settings.commandLog &&
                 storeEvents == settings.storeEvents &&
-                parallelPosting == settings.parallelPosting;
+                postEvents.equals(settings.postEvents);
     }
 
     @SuppressWarnings("NonFinalFieldReferencedInHashCode")
     @Override
     public int hashCode() {
-        return Objects.hashCode(commandLog, storeEvents, parallelPosting);
+        return hash(commandLog, storeEvents, postEvents);
```

Comment:
Obtaining the hash code of the executor is not a good idea. Generally speaking, it's better not to hash any objects with a volatile state (executors, network connections, etc). There is no guarantee that their hash code value is consistent through their lifespan, and does not depend on their changing state.